src/app/features/course/course.component.html

```diff
@@ -1 +1,67 @@
-<p>course works!</p>
+<form class="app-course" [formGroup]="form" (ngSubmit)="onSubmit()">
+    <div class="form-group row">
+        <label for="exampleFormControlInput1">Titles</label>
+        <div class="col-10">
+            <input type="text" class="form-control" formControlName="title" id="exampleFormControlInput1"
+                placeholder="title" [ngClass]="{ 'is-invalid': submitted && title?.errors }">
+            <div *ngIf="submitted && title?.errors" class="invalid-feedback">
+                <div *ngIf="title?.errors?.['required']">
+                    title is required
+                </div>
+            </div>
+        </div>
+        <div class="form-group col-2">
+            <button type="submit" class="btn btn-primary">Create Course</button>
+        </div>
+    </div>
+    <div class="form-group">
+        <label for="exampleFormControlTextarea1">Description</label>
+        <textarea class="form-control" formControlName="description" id="exampleFormControlTextarea1" rows="6"
+            [ngClass]="{ 'is-invalid': submitted && description?.errors }"></textarea>
+        <div *ngIf="submitted && description?.errors" class="invalid-feedback">
+            <div *ngIf="description?.errors?.['required']">
+                description is required
+            </div>
+        </div>
+    </div>
+    <div class="app-add-author">
+        <div class="form-group">
+            <div class="row">
+                <div class="row col-6">
+                    <h1 class="app-author">Add author</h1>
+                    <label>Author Name</label>
+                    <input type="text" class="form-control app-input" formControlName="authorName"
+                        placeholder="author name">
+                    <app-button class="app-button" [buttonText]="'Çreate author'"></app-button>
+                    <h2 class="app-duration">Duration</h2>
+                    <div class="form-group">
+                        <label>Duration</label>
+                        <input type="text" class="form-control app-input" formControlName="duration"
+                            placeholder="duration" [ngClass]="{ 'is-invalid': submitted && duration?.errors }">
+                        <div *ngIf="submitted && duration?.errors" class="invalid-feedback">
+                            <div *ngIf="duration?.errors?.['required']">
+                                duration is required
+                            </div>
+                        </div>
+                        <h6>Duration: <span>00:00 houre</span></h6>
+                    </div>
```

Comment:
Currently `duration` block is located in the center of authors block, but duration itself doesn't depend on authors. To make borders between form parts clear, lets move `duration` template after `authors` template.

Reply:
i needed to split page in two part as it was on mockups and because of that i put author and duration in col-6 and authors list in next col-6